### ISSUE TITLE
Position of Virus tail terms 

### ISSUE THREAD

**Issue by sandyl27** [geneontology/go-ontology#16705]
Shouldn't these terms be under GO:0098019 virus tail, major subunit:
GO:0098027 virus tail, sheath- the definition should specify myophage 
GO:0098028 virus tail, shaft
GO:0098026 virus tail, tube- the definition should specify myophage and podophage

@jrr-cpt 
@jimhu-tamu

**Comment by pgaudet** [comment]
GO:0098019  virus tail, major subunit is a strange term, at least based on its definition: "The part of the viral tail that comprises the most common subunit type.". This is rather vague. The term has no annotations. 

@bmeldal did you request that term ? 

Thanks, Pascale

**Comment by pgaudet** [comment]
The same points are true regarding     GO:0098020  virus tail, minor subunit

**Comment by bmeldal** [comment]
Nope, never used it. Sounds rather vague to me, too. I only ever worked with globular viruses anyway and that was before I even heard of GO ;-)

**Comment by pgaudet** [comment]
OK - I thought GOC:bm was you :)

**Comment by pgaudet** [comment]
@sandyl27 I would propose to obsolete these two terms - is that OK for you ?

**Comment by sandyl27** [comment]
So I wouldn't obsolete them because we have a protein that is common in these bacteriophage viruses that we want to annotate to minor subunit. But I would put them as parent terms and clear up the definition and maybe make the major subunit as a higher level term that says do not annotate.

**Comment by pgaudet** [comment]
@sandyl27 Can you provide new definitions ? 

Also, can you provide some hierarchy ? It's add that the minor and major subunits are sibling of all other terms

<img width="714" alt="image" src="https://user-images.githubusercontent.com/4782928/49212360-2254fc00-f3c2-11e8-9abb-846d1ff48682.png">


@pmasson55 Any thoughts on this one ?

**Comment by jrr-cpt** [comment]
The major tail subunit refers to the protein found in highest copy number in the tail of a bacteriophage, usually determined by protein gel. This is the 'tail shaft' in a siphophage (long, flexible tail), and the 'tail sheath' for myophage (contractile tails), and and sometimes 'tail tube' for myophage and podophage (short tails). Most other proteins that comprise the tails (tip, wedge, baseplate, fibers, tape measure, etc) are considered minor tail subunits, again based on copy number per tail. Where the viruses of archaea fit into this hierarchy is unclear. I would suggest these relationships and the definitions that follow.
![virus tail go](https://user-images.githubusercontent.com/38894438/49256918-3581bf00-f3f6-11e8-961d-60dd8076abbc.JPG)

**GO:0098019 virus tail, major subunit** - Structural components of a bacteriophage tail, present at higher copy number relative to other minor tail proteins. Comment: Particularly used to describe the major subunits of the tailed bacteriophage types myophage and siphophage, and sometimes podophage. Do not annotate.
**GO:0098020 virus tail, minor subunit** - Structural components of a bacteriophage tail, found in copy number (usually 1-30) lower than the major tail proteins (sheath, shaft, or tube). Comment: Particularly used to describe the minor subunits of the tailed bacteriophage types myophage,  siphophage, or podophage.

**Comment by jrr-cpt** [comment]
Possibly a helpful paper. PMID: 24616838. There are others. 
Also the viral zone diagrams on [caudovirales](https://viralzone.expasy.org/4076) and [myo-/sipho-/podophage](https://viralzone.expasy.org/885) might be helpful. 

@sandyl27 
@jimhu-tamu

**Comment by pgaudet** [comment]
Talking with @pmasson 

A couple of points: 

1. Minor and major subunits are not orthogonal classifications to the other terms (sheath, shaft, tube). Depending on the virus/phage, it may be that the major subunit does not contain the same parts (ie in some viruses the shaft may be very long, in others the tube, for example). 
2. This is rather a biochemical definition, not a functional definition; really what information as we adding ? 
3. The paper xou cite doesn'r describe major/minor tail subunits (or did I miss the correct section ? ) 

Thanks, Pascale

**Comment by sandyl27** [comment]
So I believe @jimhu-tamu, @jrr-cpt and I have come to the conclusion that we are okay with obsoleting the terms minor and major subunit.

**Comment by pgaudet** [comment]
Thanks for the feedback @jrr-cpt 

Can you write the new definitions for GO:0098027, GO:0098028 and GO:0098026 as you would like them ? I can try but I am sure yours will sound better. 

Thanks, Pascale

**Comment by pgaudet** [comment]
The proposal has been made to obsolete

GO:0098019 virus tail, major subunit
GO:0098020 virus tail, minor subunit
The reason for obsoletion is that these are not precise structures: major (most abundant) and minor subunits of viral tails vary in different viruses/phages. 

There are no annotations to those terms. There are no mappings; this term is not present in any subsets/slims. Any comments can be  added to the issue: https://github.com/geneontology/go-ontology/issues/16705

We are opening a comment period for this proposed obsoletion. We’d like to proceed and obsolete this term on December 10th, 2018. Unless objections are received by December 10th, 2018.  we will assume that you agree to this change.

Thanks, Pascale

**Comment by paolaroncaglia** [comment]
@pgaudet
If memory serves me right, those viral terms were part of the viral mapping work carried out by Brenley McIntosh in collaboration with Jane and Becky. See the definition sources (Source: PHI:0000082, GOC:bm). I wasn't involved and can't remember the details, but I think that this publication stemmed from that work: https://www.ncbi.nlm.nih.gov/pubmed/26215368, so you may want to check briefly with at least one of the authors. Of course, if a GO term is published it doesn't mean that it can't be obsoleted, etc. :-)
Thanks,
Paola

**Comment by phagemu** [comment]
Hi everyone,
My first entry on this site! A new website of bacteriophage protein families is on its way to fill in the gap left by the absence of update of ACLAME. A historical point: the PhiGO/MeGO ontology was developed on the side of ACLAME. It was used in Geneva to define new terms that were added to GO. These include the tail phage component terms discussed here. I agree with @jimhu-tamu's comments, definition corrections and suggested "tree" but I don't understand Sandyl's conclusion that "we are okay with obsoleting the terms minor and major subunit". In other words I'm in favor of keeping the major and minor subunit terms. I think there are cases where the protein is known to be abundant in the tail but it's precise position in the tail isn't known. 
I'm presently using the GO terms to annotate the protein families in the new DB. We have retrieved all GO annotations appended to the individual proteins in the DB, which contains viral/phage  proteins and proteins from predicted prophages. Clearly these are not really informative as they almost never  refer to the viral origin of the protein (normal since those viral terms were probably not in GO when those genomes were annotated!).

**Comment by pgaudet** [comment]
Hello Ariane, 

Thanks for the comment, that helps understand how these terms were supposed to be used - unfortunately protein abundance is not in the scope of GO. In other words, if all a paper shows is that a protein is more abundant than another, there is not enough data to warrant a GO annotation.   

We look forward to your contributions to the GO project ! It'd be nice to know more about your new database. 

Thanks, Pascale

**Comment by phagemu** [comment]
Hello Pascale,
I certainly understand the point but I thought Jim’s description was clarifying the point. Knowing that a phage tail protein is « major » or "minor «  provides additional information to the description of the viral tail. If those terms are removed only viral tail will be left to describe all tail components, from the tail tape measure protein to the tube, sheath, shaft, completion, etc. for all phages for which the 3D  structure of the tail hasn't been solved, which means 99% of the phages sequenced today. The same matter may arise for capsid components. There are also major and minor components there… Looking forward to more comments… 

Ariane Toussaint
Génétique et Physiologie Bactérienne
Université Libre de Bruxelles 
IBMM-DBM 
12 Rue des Professeurs Jeneer et Brachet 
B 6041 Gosselies, Belgium 
Tel: +3223546290
Cell: +32472700183

**Comment by phagemu** [comment]
This is not absolutely correct because even in different overall tail structures (e.g. flexible and contractile) there are proteins related by their sequence or 3D structure. Tape measure proteins and the completion proteins are present in both for instance. And there are bacterial components that are related to some of the flexible phage tails, the type VI/6 secretion system… They have major and minor proteins too! Horizontal transfer is ubiquitous in these organisms. For a phage person at least this it is biologically meaningful! Hope other phage addicts could enter this discussion...

Ariane Toussaint
Génétique et Physiologie Bactérienne
Université Libre de Bruxelles 
IBMM-DBM 
12 Rue des Professeurs Jeneer et Brachet 
B 6041 Gosselies, Belgium 
Tel: +3223546290
Cell: +32472700183

**Comment by SIBvirus** [comment]
Hello,
I agree with the proposal to obsolete these two terms. We also used these terms in Swiss-Prot when naming phage proteins in the past, however the terms "minor" and "major" do not give any functional information and I replace them as soon as I know the precise function of the tail protein. Logically, if we tag the major tail protein, all other tail proteins would be "minor tail proteins", which is rather uninformative...
I prefer to use the precise terms (virus tail fiber, virus tail tube, virus tail sheath, etc..), which we could even extend (?add virus tail, completion; virus tail, tape measure?). In the case of tail proteins that would have been quantified but not further studied (a rare case I think!), we can still use the "virus tail" term.
Hope this helps,
Chantal Hulo
Swiss-Prot/Uniprot Virus program

**Comment by phagemu** [comment]
Hi again,
It’s right that beside tube, sheath and shaft other tail proteins are minor. I thought again about al this over the WE and, inspired by Jim’s schema I wondered if we could divide long and short tails and further divide  long tails into flexible and contractile ones. I think it would definitely by useful to add tail tape measure and tail completion terms, because we have several families of these in the new DB and otherwise they would all come under virus tail. Chantal I have a question about podoviral tails. Did we discuss specific terms for those with Ian Molineux ? It’s  complicated since they are head proteins that become tail ones… how would you deal with this ?
If everyone agrees that the two tail terms should be removed, I guess we can still use them in our DB annotations (we have 40000 major proteins grouped into 12 families, and 21000 minor ones in 150 families!), use the GO viral tail for those and, as you do replace them later, when more is known. 
I fear that capsid major and minor subunits might come into such a discussion as well since, at least among the minor subunits there are also proteins with different functions (we have decoration but there are the completion protein, the penton, spike etc.) 
Maybe the present discussion could be an opportunity to revisit the viral terms and their hierarchy in more details. Just a  few things I had on my list of things to do: 
- use capsomer instead of capsid major subunit? 
- Add (MF, CC and BP mixed up) 
   - "negative regulation by virus of host cell nuclease » a process widely present in viruses, 
   - « regulation of viral transcription » because we have « negative regulation of viral transcription" and « positive regulation of viral transcription » and there are regulators  that could potentially be one or the other or that have both functions (or should one use both terms in this case ?), 
   - « viral sigma factor activity », and "lytic viral release » since the "non lytic viral release » term exists GO:0046753. 
To improve the relationships between viral terms I need to spend more time on the matter. If I know that GO is interested I can do it. And we can provide definitions  synonyms etc. for the terms to be added if necessary.  Have a nice day.

Ariane Toussaint
Génétique et Physiologie Bactérienne
Université Libre de Bruxelles 
IBMM-DBM 
12 Rue des Professeurs Jeneer et Brachet 
B 6041 Gosselies, Belgium 
Tel: +3223546290
Cell: +32472700183

**Comment by jimhu-tamu** [comment]
Hi Ariane (love the username @phagemu, by the way). Good to hear from you!

Responding to different parts:

> I think it would definitely by useful to add tail tape measure and tail completion terms, because we have several families of these in the new DB and otherwise they would all come under virus tail. Chantal I have a question about podoviral tails. Did we discuss specific terms for those with Ian Molineux ? It’s  complicated since they are head proteins that become tail ones… how would you deal with this ?

I agree on adding more terms and we're working on that. The general solution in my mind is to make more terms for distinct functions within the scope of GO. For me, the qualitative stoichiometry of being a major or minor subunit of heads or tails based on staining a gel of virions, capsids, or tails is a form of evidence that we use to infer that a particular gene product is part of a particular virion component.

> Chantal I have a question about podoviral tails. Did we discuss specific terms for those with Ian Molineux ? It’s  complicated since they are head proteins that become tail ones… how would you deal with this ?

I'm not sure what you mean about head proteins that become tail ones. Could you elaborate? I've had some discussions with Ry about heads and tails and podophages and we clearly need to think about how to handle them... and talk to Ian and others.

> I fear that capsid major and minor subunits might come into such a discussion as well since, at least among the minor subunits there are also proteins with different functions (we have decoration but there are the completion protein, the penton, spike etc.)

In fact, I think major and minor capsid protein should also be outside the scope of GO for similar reasons, but that there should be terms for penton, hexon, spike, etc. 

Finally, we (GOC) just started a project to reexamine representation of multi-organism biology in GO. In part this was driven by phage biology, but the scope will be much larger than phage and viruses.

**Comment by selewis** [comment]
Just curious, is there a viral 'anatomy' ontology? Things maybe outside the scope of GO, but they should have a home. @cmungall do you know? or @jimhu-tamu? or anyone?

**Comment by phagemu** [comment]
Hi Jim, 
Good to read from you again. I obviously was not clear in my « head » or my purpose when I wrote about podo tails: I should have referred to ejection of course… I’m not enough into the GO logic to automatically distinguish between processes and components… I mixed up the tail, its dynamics and its consequences. 
Now that I have access to a sort of updated ACLAME (not public yet, because we still need to check the website functionality and coherence of our annotations!), I’d be ready to help in listing the terms that everyone agrees should be added to or removed from the viral ontology.  Could we go for CC and BP? It would be good to have SIB contribution as well...

Ariane Toussaint
Génétique et Physiologie Bactérienne
Université Libre de Bruxelles 
IBMM-DBM 
12 Rue des Professeurs Jeneer et Brachet 
B 6041 Gosselies, Belgium 
Tel: +3223546290
Cell: +32472700183

**Comment by jimhu-tamu** [comment]
@phagemu: I agree that we need more development of the phage biology in component, process, and molecular function aspects!

**Comment by SIBvirus** [comment]
Hello!
It would be really great to extend the phage nomenclature in GO!! I fully agree to be more precise and describe as much as possible all the different parts of tail and capsid. Ready to help!
Regarding the podoviridae, we annotated the internal head proteins that are ejected neither as capsid protein (we keep this term for the proteins that form the icosahedron) nor as tail protein. However, this leads to your second question Ariane:
We created the following terms to describe the type of ejection when working with Ian Molineux (biological process):
GO:0099000: viral genome ejection through host cell envelope, contractile tail mechanism   
GO:0099001: viral genome ejection through host cell envelope, long flexible tail mechanism 
GO:0099002: viral genome ejection through host cell envelope, short tail mechanism

These terms allow to tag all the proteins involved in the ejection process, even those which are not part of the capsid or the tail. They can be combined with structural (component) terms.
